lib/SimpleSchema.js

```diff
@@ -794,14 +801,35 @@ class SimpleSchema {
  * PRIVATE
  */
 
+function mergeSchemas(schemas) {
+  const mergedSchema = {};
+  forEach(schemas, schema => {
+    // Loop through and extend each individual field
+    // definition. That way you can extend and overwrite
+    // base field definitions.
+    forEach(schema, (def, field) => {
+      mergedSchema[field] = mergedSchema[field] || {};
+      if (!(mergedSchema[field] instanceof SimpleSchemaGroup)) {
+        if (def instanceof SimpleSchemaGroup) {
+          mergedSchema[field] = def;
+        } else {
+          Object.assign(mergedSchema[field], def);
+        }
+      }
+    });
+  });
+  return mergedSchema;
+}
+
```

Comment:
You can delete the `mergeSchemas` function. I think that was something I originally left in by mistake after converting this from the Meteor package, and then removed after this PR was branched.